sunpy/lightcurve/__init__.py

```diff
@@ -9,6 +9,14 @@
 __author__ = "Keith Hughitt"
 __email__ = "keith.hughitt@nasa.gov"
 
+import warnings
```

Comment:
why has this just appeared?

Reply:
I didn't see adep warning for lightcurve in the __init__ like we have for WCS.

Reply:
I have a vague recollection that was an active choice for 0.8. I think it's worth having for 0.9.